packages/neos-ui-editors/src/EditorEnvelope/index.js

```diff
@@ -62,6 +75,10 @@ export default class EditorEnvelope extends PureComponent {
     }
 
     render() {
+        if (this.state.error) {
+            return <div>{this.state.error && this.state.error.toString()}</div>;
```

Comment:
Double test for this.state.error, no ?

Reply:
Any possibility to have an empty string here ? So maybe a default message can help ?